ion-schema-tests-runner/tests/ion-schema-tests-2-0.rs

```diff
@@ -2,14 +2,12 @@ use ion_schema_tests_runner::ion_schema_tests;
 
 ion_schema_tests!(
     root = "ion-schema-tests/ion_schema_2_0/",
-    // Support for ISL 2.0 is not implemented yet, so all tests are ignored.
+    // Support for ISL 2.0 is not implemented yet, so some tests are ignored.
```

Comment:
If we're being pedantic... 😉 
```suggestion
    // Support for ISL 2.0 is not completely implemented yet, so some tests are ignored.
```